pkg/apis/crd/v1alpha1/types.go

```diff
@@ -432,6 +432,10 @@ type Rule struct {
 	// when rules are matched. Should be default to false.
 	// +optional
 	EnableLogging bool `json:"enableLogging"`
+	// LogLabel is a user-defined arbitrary string which will be printed in the
+	// NetworkPolicy logs and exported in flow records.
```

Comment:
As we do not support adding the label to flow records yet, at least do not say that in the comments in YAMLs.

Reply:
Updated.